website/public/js/controllers/challengesCtrl.js

```diff
@@ -116,11 +114,14 @@ habitrpg.controller("ChallengesCtrl", ['$rootScope','$scope', 'Shared', 'User',
         rewards: clonedTasks.reward,
         leader: User.user._id,
         group: challenge.group._id,
-        timestamp: +(new Date),
-        members: [],
```

Comment:
These two get populated automatically when the challenge is saved, no need to initialize them.